roles/network_plugin/multus/tasks/main.yml

```diff
@@ -10,12 +10,28 @@
     - {name: multus-clusterrole, file: multus-clusterrole.yml, type: clusterrole}
     - {name: multus-clusterrolebinding, file: multus-clusterrolebinding.yml, type: clusterrolebinding}
   register: multus_manifest_1
+  when: inventory_hostname == groups['kube_control_plane'][0]
+
+- name: Multus | Check container engine type
+  set_fact:
+    container_manager_types: "{{ ansible_play_hosts_all|map('extract', hostvars, ['container_manager'])|list|unique }}"
+  run_once: true
+  when: inventory_hostname == groups['kube_control_plane'][0]
 
 - name: Multus | Copy manifest templates
```

Comment:
This task was failed like https://gitlab.com/kargo-ci/kubernetes-sigs-kubespray/-/jobs/2957405780#L3428
```
TASK [network_plugin/multus : Multus | Copy manifest templates] ****************
task path: /builds/kargo-ci/kubernetes-sigs-kubespray/roles/network_plugin/multus/tasks/main.yml:21
Wednesday 31 August 2022  05:09:30 +0000 (0:00:00.114)       0:11:25.330 ****** 
Wednesday 31 August 2022  05:09:30 +0000 (0:00:00.114)       0:11:25.330 ****** 
fatal: [instance-2 -> {{ groups['kube_control_plane'][0] }}]: FAILED! => {"msg": "The conditional check 'item.engine in container_manager_types' failed. The error was: error while evaluating conditional (item.engine in container_manager_types): 'container_manager_types' is undefined\n\nThe error appears to be in '/builds/kargo-ci/kubernetes-sigs-kubespray/roles/network_plugin/multus/tasks/main.yml': line 21, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Multus | Copy manifest templates\n  ^ here\n"}
```
The root problem seems `'container_manager_types' is undefined`
At the above line 17, container_manager_types is set only if `inventory_hostname == groups['kube_control_plane'][0]`
So we might need to add the same condition to the line 35.